libkbfs/interfaces.go

```diff
@@ -1502,11 +1503,15 @@ type BareRootMetadata interface {
 	// folder.  This is only expected to be set for folder resets.
 	IsFinal() bool
 	// IsWriter returns whether or not the user+device is an authorized writer.
-	IsWriter(user keybase1.UID, deviceKID keybase1.KID) bool
+	// Note "wkb" is expected to be null for pre-v3 metadata.
+	IsWriter(user keybase1.UID, deviceKID keybase1.KID, wkb *TLFWriterKeyBundleV2) bool
```

Comment:
Oh, and since it's named `ExternalMetadata`, we can probably just mention in BRMDV2 comments that, since it embeds everything, it will always expect that parameter to be nil. Then we don't have to mention it in the interface comments here.